docs/FAQ.md

```diff
@@ -26,6 +26,7 @@ Summarize the question and quote the reply, linking back to the original comment
 * [How does `dep` handle symbolic links?](#how-does-dep-handle-symbolic-links)
 * [Does `dep` support relative imports?](#does-dep-support-relative-imports)
 * [How do I make `dep` resolve dependencies from my `GOPATH`?](#how-do-i-make-dep-resolve-dependencies-from-my-gopath)
+* [How do I use `dep` with Docker?](#how-do-i-make-dep-resolve-dependencies-from-my-gopath)
```

Comment:
The link should point to `#how-do-i-use-dep-with-docker`.

Reply:
Done.